node/tessel-export.js

```diff
@@ -44,8 +44,8 @@ function Tessel(options) {
   }
 
   this.ports = {
-    A: options.ports.A ? new Tessel.Port('A', '/var/run/tessel/port_a', this) : null,
-    B: options.ports.B ? new Tessel.Port('B', '/var/run/tessel/port_b', this) : null,
+    A: options.ports.A ? new Tessel.Port('A', Tessel._socketPaths.A, this) : null,
+    B: options.ports.B ? new Tessel.Port('B', Tessel._socketPaths.A, this) : null,
```

Comment:
A couple things here: 

- These aren't defined anywhere in this file.
- Both ports use `Tessel._socketPaths.A`
- I think it's more appropriate to define these on `Tessel.Port`, as `Tessel.Port.PATH.A` & `Tessel.Port.PATH.B`